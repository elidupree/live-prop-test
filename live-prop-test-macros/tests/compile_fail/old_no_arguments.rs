use live_prop_test::live_prop_test;

#[live_prop_test(postcondition = "old()")]
fn five() -> i32 {
  5
}

fn main() {}
