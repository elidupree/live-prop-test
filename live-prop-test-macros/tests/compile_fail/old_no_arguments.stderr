error: `old` requires an expression as its "argument"
 --> $DIR/old_no_arguments.rs:3:34
  |
3 | #[live_prop_test(postcondition = "old()")]
  |                                  ^^^^^^^
